packages/flutter_tts/tizen/src/text_to_speech.cc

```diff
@@ -109,13 +121,25 @@ std::vector<std::string> &TextToSpeech::GetSupportedLanaguages() {
   return supported_lanaguages_;
 }
 
-tts_state_e TextToSpeech::GetState() {
+TtsState TextToSpeech::GetState() {
   tts_state_e state;
   int ret = tts_get_state(tts_, &state);
   if (ret != TTS_ERROR_NONE) {
-    LOG_ERROR("[TTS] tts_prepare failed: %s", get_error_message(ret));
+    LOG_ERROR("[TTS] tts_get_state failed: %s", get_error_message(ret));
+    return TtsState::kError;
```

Comment:
It does not seem appropriate for API call failures turn into the state of TTS.

Reply:
done